src/measure_theory/measure_space.lean

```diff
@@ -1444,8 +1436,37 @@ instance probability_measure.to_finite_measure (μ : measure α) [probability_me
 lemma probability_measure.ne_zero (μ : measure α) [probability_measure μ] : μ ≠ 0 :=
 mt measure_univ_eq_zero.2 $ by simp [measure_univ]
 
+lemma prob_add_compl_eq_one [probability_measure μ]
```

Comment:
```suggestion
lemma prob_add_prob_compl [probability_measure μ]
```
Since you're adding two probabilities, the lemma should probably be named like this to ensure maximal discoverability, i.e., correspond more or less algorithmically to the sequence of symbols in the statement of the lemma. The `eq_one` is not really useful, as what else could it be?

Reply:
The lemma above uses the name `probability_measure.ne_zero`. I think we should decide whether to use `probability_measure.` or `prob_` as a prefix, and use it uniformly

Reply:
It's not the same in the two lemmas: for the other one, it is saying that the measure is nonzero. Here, it is saying that the measure of some set (most often called the probability of this set) is nonzero. I think my preference would be to define a real-valued version of the measure and use `prob` in lemma statements for this real-valued version.